airflow/providers/common/sql/operators/sql.py

```diff
@@ -385,17 +385,15 @@ def execute(self, context: 'Context'):
         self.sql = f"SELECT check_name, check_result FROM ({checks_sql}) "
         f"AS check_table {partition_clause_statement};"
 
-        records = hook.get_pandas_df(self.sql)
+        records = hook.get_records(self.sql)
```

Comment:
Did you forget to commit something? Tests fail:

> \>      records = hook.get_records(self.sql)
> E       AttributeError: 'MockHook' object has no attribute 'get_records'

Reply:
No, I just failed to run the rest of the tests (I only checked my new test that hits a _real_ DB that I added in a paralell PR not the mocked one.)